src/views/tickets.js

```diff
@@ -0,0 +1,163 @@
+import React, { Component } from 'react';
+import BootstrapTable from 'react-bootstrap-table-next';
+import paginationFactory from 'react-bootstrap-table2-paginator';
+import UserContext from '../UserContext';
+import { Link } from "react-router-dom"
+import Accordion from '../components/Accordion';
+import * as axios from 'axios';
+
+const columns = [{
+    dataField: 'name',
+    text: 'Ticket',
+    headerStyle: () => {
+      return { width: "20%" };
+    }
+  }, {
+    dataField: 'propertyManager',
+    text: 'Sender',
+    sort: true,
+    headerStyle: () => {
+      return { width: "20%" };
+    }
+  }, {
+    dataField: 'address',
+    text: 'Assigned To',
+    sort: true,
+    headerStyle: () => {
+      return { width: "20%" };
+    }
+  }, {
+    dataField: 'tenants',
+    text: 'Created',
+    sort: true,
+    headerStyle: () => {
+      return { width: "20%" };
+    }
+  }, {
+    dataField: 'dateAdded',
+    text: 'Updated',
+    sort: true,
+    headerStyle: () => {
+      return { width: "20%" };
+    }
+  }];
+
+const pageButtonRenderer = ({
+  page,
+  active,
+  disable,
+  title,
+  onPageChange
+}) => {
+  const handleClick = (e) => {
+    e.preventDefault();
+    onPageChange(page);
+  };
+  const activeStyle = {};
+  if (active) {
+    activeStyle.backgroundColor = 'black';
+    activeStyle.color = 'white';
+  } else {
+    activeStyle.backgroundColor = 'gray';
+    activeStyle.color = 'black';
+  }
+  if (typeof page === 'string') {
+    activeStyle.backgroundColor = 'white';
+    activeStyle.color = 'black';
+  }
+  return (
+    <li className="page-item">
+      <a href="#" onClick={ handleClick } style={ activeStyle }>{ page }</a>
+    </li>
+  );
+};
+
+const options = {
+  pageButtonRenderer
+};
+
+export class Tickets extends Component {
+    constructor(props) {
+        super(props);
+        
+        this.state = {
+        tickets: [],
+        }
+
+        this.getTickets = this.getTickets.bind(this)
+    }    
+
+    componentDidMount() {
+        this.getTickets(this.context);
+    }
+
+    getTickets = (context) => {
+        axios.get(`${process.env.REACT_APP_API_URL}/properties`, { headers: {"Authorization" : `Bearer ${context.user.accessJwt}`} })
```

Comment:
Note the axios requests will need to look like `axios.get("/api/properties", ...)` (see https://github.com/codeforpdx/dwellingly-app/pull/177)